### ISSUE TITLE
Does not open the portal on both windows and Mac

### ISSUE THREAD

**Issue by DarioNItti94** [FluxionNetwork/fluxion#1144]
#### [Summary]([url](url))
every time i try to create a portal and try to get a windows or mac device to connect, the devices connect but they don't automatically open the portal. This is my log:

![Schermata del 2023-03-17 18-31-08](https://user-images.githubusercontent.com/26146240/225980345-852b2a21-1e02-4380-a866-66d3bf04cf0f.png)

**Comment by debiandragon** [comment]
do devices get an ip address ? my problem is the devices connect but never get ip address from fluxion server = password request page never pops up

**Comment by yamix007** [comment]
> do devices get an ip address ? my problem is the devices connect but never get ip address from fluxion server = password request page never pops up

yes, same the trick is on isc & hostpad, i don't advise you mess up with the settings or you'll have to install everything from scratch to work for you again

i tried to edit the dhcp & hostapd files for that but still not showing up automatically ...

if anybody can help, thanks in advance !
